File: dataframes/SelectColumns.jl
module SelectColumns

using DataFrames, Query

################
## DataFrames ##
################

name_treat = select(person, :name, :fav_treat)


###########
## Query ##
###########

# Query - stand alone
name_treat2 = person |>
    @select(:name, :fav_treat) |>
    DataFrame

# Query - LINQ
name_treat3 = @from i in person begin
    @select {i.name, i.fav_treat}
    @collect DataFrame
end

end # module

File: dataframes/selectcolumns.R
############
## base R ##
############

name_treat <- person[,c("name", "fav_treat")]
name_treat2 <- subset(person, select=c("name", "fav_treat"))


###########
## dplyr ##
###########

library(dplyr)

name_treat3 <- person %>%
  select(name,
         fav_treat)
